### ISSUE TITLE
Package.loadSettings is deprecated.

### ISSUE THREAD

**Issue by raaka1** [lee-dohm/language-r#3]
Store package settings files in the `settings/` directory instead of `scoped-properties/`
```
Package.loadSettings (/Applications/Atom.app/Contents/Resources/app/src/package.js:565:9)
<unknown> (/Applications/Atom.app/Contents/Resources/app/src/package.js:187:43)
```

**Comment by lee-dohm** [comment]
Thanks for reporting!

**Comment by RaviKiran-IO** [comment]
Thanks looking forward for upgrade

**Comment by Chris1221** [comment]
This popped up for me on v0.190.0 unfortunately.  Wasn't present on 0.189 or previous (I'm almost certain). Thanks for a great package by the way, use it every day.

**Comment by lee-dohm** [comment]
Fixed in v0.2.0 :grinning:

**Comment by raaka1** [comment]
Thanks this is why, i love GitHub
